### PR TITLE
Docs improvement for 0.16.0 preview release batch six

### PR DESCRIPTION
**Pull request by SteveMutungi254**
Docs improvement for 0.16.0 preview release batch six

### CODE REVIEW

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaUser.md [hunk 1]
```diff
@@ -34,13 +33,19 @@ Remove-EntraBetaUser
 
 The `Remove-EntraBetaUser` cmdlet removes a user in Microsoft Entra ID. Specify the `ObjectId` parameter to remove the specified user in Microsoft Entra ID.
 
+The calling user must be assigned at least one of the following Microsoft Entra roles:
+
+- User Administrator
+
```
Comment (KenitoInc) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Remove-EntraBetaUser.md:39]:
```suggestion
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraTrustedCertificateAuthority.md [hunk 2]
```diff
@@ -45,14 +44,17 @@ $cer = Get-EntraTrustedCertificateAuthority #Get the CertificateAuthorityInforma
 Remove-EntraTrustedCertificateAuthority -CertificateAuthorityInformation $cer[0]
 ```
 
-```output
+```Output
```
Comment (KenitoInc) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraTrustedCertificateAuthority.md:47]: We need consistency.
I see `Output` starts with a capital letter then in other areas I see `powershell` starting with small letter

Comment (SteveMutungi254) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraTrustedCertificateAuthority.md:47]: That is the design: Output should always start with capital O and powershell is all small letters. That is based on the general recommendation from the core PowerShell docs team.

Comment (alexandair) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraTrustedCertificateAuthority.md:47]: The language tag is not case-sensitive, but you should use lowercase except for a few special cases.

Code fences without tags can be used for syntax blocks or other types of content where syntax highlighting is not required.

When showing output from a command, use a tagged code fence with the language tag Output. The rendered box is labeled as Output and doesn't have syntax highlighting.

- From the PowerShell docs

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**